docs/release.md

```diff
@@ -23,36 +23,16 @@ if the version specified is not the one you want to release, then make a PR to u
 2. Run release workflow in the [github actions release page](https://github.com/scality/vaultclient-java/actions/workflows/release.yml).
 
 
-3. Once the release workflow is complete successfully, proceed to Nexus Repository Manager to manually release the project.
-   > We are doing this as currently OSIS and vaultlcient-java share the same namespace and to automate closing and 
-   > releasesing all previous versions should be closed and released. Will be improved in the future. 
-   
-    Nexus Repo link: https://oss.sonatype.org
-
-    For login credentials, they are stored in github secrets or can be obtained by emailing Object Squad <object-squad@scality.com>
+3. Once the release workflow is complete successfully, proceed to Nexus Repository Manager to check if the released package is present.
     
-   
-
-4. Go to [Staging Repositories](https://oss.sonatype.org/#stagingRepositories), 
-find the latest record which should correspond to the github action release workflow time
-   
-   ![](./pictures/staging-repositories.png)
-
-
-5. Click **Close** button, fill in **Description** and click **Confirm** to start the Close process.
-
-   ![](./pictures/close-confirmation.png)
-
-
-6. Once Close process is complete successfully, you should be able to see the status of 
-this record is closed and the **Release** button is active.
-
-   ![](./pictures/staging-repositories-2.png)
-
-
-7. Click **Release** button, fill in **Description** and click ***Confirm** to start the Release process.
-
-   ![](./pictures/release-confirmation.png)
+   > Nexus Repo link: https://oss.sonatype.org/#nexus-search;quick~com.scality
+   > 
+   > For login credentials, they are stored in github secrets or can be obtained by emailing Object Squad <object-squad@scality.com>
+    
+4. If the release workflow fails at ``Gradle Publish Release to Nexus`` step, and raises error `Wrong number of received repositories in state 'open'. Expected 1, received 2`.
 
+   >That's because that there are old open stating repositories from this project or OSIS project that has not yet been released. 
```

Comment:
Can you quickly explain this sentence to me? Do you mean staging? Maybe:
"It's because there are old open staging repositories from this project or OSIS project that have not been released yet."

Reply:
this was indeed not very clear 😅 
[this step gradle uploadArchive](https://github.com/scality/vaultclient-java/pull/28/files#diff-c638d5d32fa62ee9cf43dd38cd5b37e8d12652857c685a289ec747f66065ea51R49) is send the packages to a place named `staging repositories` in Nexus, and the packages will be staying there open if we don't execute [the next step gradle closeAndRelease ](https://github.com/scality/vaultclient-java/pull/28/files#diff-c638d5d32fa62ee9cf43dd38cd5b37e8d12652857c685a289ec747f66065ea51R53) to release them. so if sometime we do a `grade uploadArchive` manually and forget to gradle closeAndRrelease it and forget to release it manually from the staing repositories, whenever  we want to release another package, gradle will raise this error, it doesn't know which one to release, it always expected that there is only one open repository waiting to be released.